packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -20,11 +22,11 @@ export function buildRootObject(vatPowers) {
       createRepeater(delaySecs, interval) {
         assert(
           Nat(delaySecs) >= 0,
-          details`createRepeater's first parameter must be a non-negative integer. ${delaySecs}`,
+          details`createRepeater's first parameter must be a non-negative integer: ${delaySecs}`,
         );
```

Comment:
This assert can never fail. Rather, of `delaySecs` is not a Nat, `Nat(delaySecs)` will throw and `assert` will never be called.